iiif.vcl

```diff
@@ -28,6 +28,7 @@ sub vcl_recv {
     set req.http.X-IIIF-Info = "true";
     set req.http.X-IIIF-Prefix = re.group.1;
     set req.http.X-IIIF-Identifier = re.group.2;
+    set req.http.Accept = if(req.http.Accept ~ "application/ld\+json|\*/\*", "application/ld+json", "application/json");
```

Comment:
Usually defer this to examples out in the wild and what :
```
▶ curl --header "Accept: */*" https://view.nls.uk/iiif/7443/74438561.5/info.json -sSI | grep -Fi Content-Type
Content-Type: application/ld+json

▶ curl --header "Accept: */*" https://dlcs.io/iiif-img/wellcome/1/ff2085d5-a9c7-412e-9dbe-dda87712228d/info.json -sSI | grep -Fi Content-Type
content-type: application/ld+json

▶ curl --header "Accept: */*" https://iiif.bodleian.ox.ac.uk/iiif/image/e6abb503-9c0c-4176-997f-3c3158a5c700/info.json -sSI | grep -Fi Content-Type
content-type: application/ld+json; charset=utf-8
```

Interestingly though, JSON is used more commonly for other parts of IIIF stores (regardless of Accept)
```
▶ curl --header "Accept: */*" https://wellcomelibrary.org/iiif/b24977160/manifest -sSI | grep -Fi Content-Type
Content-Type: application/json; charset=utf-8
```

Reply:
It does seem to imply though that this is incorrect behaviour and `application/json` by default and `application/ld+json` if in the `Accept`.

Reply:
Hmm, might need to seek clarification.

Reply:
Reading it again, those examples do look wrong. The spec reads like how Chrome advertises support for `image/webp`.

In 3.0.0-alpha, it reads:

> If the server receives a request with one of the content types above in the Accept header, it *should* respond with that content type following the rules of content negotiation. Otherwise, it *must* respond with the application/json content type.

Which appears the same. The current working copy seems to have been [prefixed with some text](https://github.com/IIIF/api/pull/1652) that confuse the issue further.

Reply:
Opened https://github.com/IIIF/api/issues/1685.